py-polars/polars/dataframe/frame.py

```diff
@@ -1235,6 +1236,26 @@ def __dataframe__(
             )
         return self.to_arrow().__dataframe__(nan_as_null, allow_copy)
 
+    def __dataframe_consortium_standard__(
+        self, *, api_version: str | None = None
+    ) -> Any:
+        """
+        Provide entry point to the Consortium DataFrame Standard API.
+
+        This is developed and maintained outside of polars.
+        Please report any issues to https://github.com/data-apis/dataframe-api-compat.
+        """
+        if _DATAFRAME_API_COMPAT_AVAILABLE:
+            from dataframe_api_compat import polars_standard  # type: ignore[import]
+        else:
+            raise ModuleNotFoundError(
+                "`dataframe-api-compat` package is required for using the "
+                "Consortium DataFrame Standard API."
+            ) from None
```

Comment:
Unless you want a specific error message, you could rely on the lazy loading to raise an error. That would simplify this code to
```python
from polars.dependencies import dataframe_api_compat
return dataframe_api_compat.polars_standard.convert_to_standard_compliant_dataframe(
            self, api_version=api_version
        )
```

Reply:
that's much simpler, thanks @zundertj !

the import is lazy, so it's OK to put it at the top of the file, right?

Reply:
Yes it is.